service/src/main/java/org/folio/rest/workflow/service/WorkflowEngineService.java

```diff
@@ -267,21 +267,29 @@ private Workflow sendWorkflowRequest(WorkflowDto workflow, String requestPath, S
         }
       }
     } catch (Exception e) {
+      log.error("Failed to send workflow request: {}!", e.getMessage(), e);
```

Comment:
Because it is still a generic exception and should be a specific exception to be able to handle with exception handler design. I would propose planning be involved in API exception handling and responses and the module descriptor updated accordingly. I think this might require JsonSchema of the exception classes.